pkg/search/proxy/recording_connector_test.go

```diff
@@ -0,0 +1,31 @@
+package proxy
+
+import (
+	"context"
+	"net/http"
+
+	"k8s.io/apiserver/pkg/endpoints/request"
+
+	"github.com/karmada-io/karmada/pkg/search/proxy/framework"
+)
+
+type recordingPlugin struct {
+	Delegate framework.Plugin
+	Called   bool
+}
+
+var _ framework.Plugin = (*recordingPlugin)(nil)
+
+func (r *recordingPlugin) Order() int {
+	return r.Delegate.Order()
+}
+
+func (r *recordingPlugin) SupportRequest(requestInfo *request.RequestInfo) bool {
```

Comment:
use the same params as `Connect`

Reply:
Connect Parameter has `rest.Responder` & `*http.Request`. It's inappropriate to send this parameters to `SupportRequest`